src/tasks/Microsoft.NET.WebAssembly.Webcil/Microsoft.NET.WebAssembly.Webcil.csproj

```diff
@@ -17,7 +17,7 @@
   <ItemGroup>
```

Comment:
This is missing a TFI == .NETFramework condition. All of these libs are inbox on .NETCoreApp.